docs/content/operations/multitenancy.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc_page
+---
+# Multitenancy Considerations
+
+Druid is often used to power user-facing data applications and has several features built in to better support high 
+volumes of concurrent queries.
+
+## Parallelization Model
+
+Druid's fundamental unit of computation is a [segment](../design/segments.html). Nodes scan segments in parallel and a 
+given node can scan `MIN(druid.processing.numThreads, Runtime.getRuntime().availableProcessors())` concurrently. To 
```

Comment:
the segments will still be scanned concurrently, just not all in parallel. I think the MIN thing is gonna be confusing to people, since it makes it sound like there's something more complicated going on than there really is.